templates/node/package.json.twig

```diff
@@ -11,7 +11,8 @@
   },
   "devDependencies": {},
   "dependencies": {
-    "request": "^2.88.0",
-    "request-promise-native": "^1.0.7"
+    "node-fetch": "^2.6.0",
+    "urljoin": "^0.1.5",
+    "form-data": "^3.0.0"
```

Comment:
Minor: Please keep the dependency list sorted alphabetically (otherwise they'll get reordered/moved around the next time someone adds/removes a dependency).